.github/actions/build-image/action.yaml

```diff
@@ -0,0 +1,58 @@
+name: build-image
+description: "Builds and pushes a docker image"
+
+inputs:
+  registry:
+    description: "Image registry to push image to"
+    required: true
+    default: ghcr.io
+  registry-username:
+    description: "Username to authenticate against image registry"
+    required: true
+  registry-password:
```

Comment:
funktioniert das so?

Passwörter müssen anders Übertragen werden. https://docs.github.com/de/actions/sharing-automations/reusing-workflows#using-inputs-and-secrets-in-a-reusable-workflow


```yml
    secrets:
      registry-password:
        required: true
 ```

Reply:
Also only works like that for workflows and not actions if i unterstand right.